### DISCUSSION TITLE
Section 3: fundme, how to add kovan testnet to metamask?

### DISCUSSION THREAD

**Discussion by satyaorz** [Cyfrin/foundry-full-course-cu#2806] [Q&A] [answered]
I am trying to add kovan using ChainList, and while searching it only shows "[Optimism Kovan](https://chainlist.org/chain/69)". so i tried to add that to my metamask but my metamask shows an error saying "Error while connecting to the custom network."
how do i fix this? i also tried it using the rpc url but same issue.

**Comment by EngrPips** [comment, marked as answer]
Hello @butterpaneermasala, kovan is deprecated. Please use `Sepolia` instead

**Reply by satyaorz** [reply]
alright, thank you.
